packages/medusa/src/api/routes/admin/orders/__tests__/cancel-fulfillment-claim.js

```diff
@@ -0,0 +1,92 @@
+import { IdMap } from "medusa-test-utils"
+import { request } from "../../../../../helpers/test-request"
+import { FulfillmentServiceMock } from "../../../../../services/__mocks__/fulfillment"
+
+describe("POST /admin/orders/:id/claims/:claim_id/fulfillments/:fulfillment_id/cancel", () => {
```

Comment:
These tests ensure that the endpoint only allow you to cancel fulfillments which are related to the given claim id, which must be related to the given order id. This pattern is repeated for the other `cancel-fulfilllment-<obj>.js` tests and implementations